code/modules/admin/verbs/config_helpers.dm

```diff
@@ -0,0 +1,16 @@
+/// Verbs created to help server operators with generating certain config files.
+
+/client/proc/generate_job_config()
+	set name = "Generate Job Configuration"
+	set category = "Server"
+	set desc = "Generate a job configuration (jobconfig.toml) file for the server. If TOML file already exists, will re-generate it based off the already existing config values. Will migrate from the old jobs.txt format if necessary."
+
+	if(!check_rights(R_SERVER))
+		return
+
+	if(SSjob.generate_config(usr))
```

Comment:
Can you add an extra input prompt to this, also letting the admin know that if they're not the host, they probably don't want this?